data/hai/unfettered side missions.txt

```diff
@@ -1426,3 +1427,132 @@ mission "Unfettered: Ambush"
+			choice
+				`	(Approach her.)`
+				`	(Get on with my business.)`
+					goto business
+
+			`	Your approach startles her out of her distraction and she pulls the boy close. When you speak to her it's clear she doesn't understand you. You look around for an Unfettered who could translate, but she grabs your sleeve and shakes her head. Eventually one of your crew who understands a little Hai gives it a try.`
+			`	"They have told me my husband is dead. The theiving raptors killed him. And now the boy has a medal but no father. They gave me his death money but there is no food and I don't know what we will do."`
+			`	You know that the Hai send food aid to the Unfettered and you've not seen anyone obviously hungry in Hai space. They would probably take this family in.`
+			choice
+				`	(Offer to take them to <destination>.)`
+				`	(Offer your condolences and move on.)`
+					goto "condolences"
+			`	While it's awkard to wait for your crew to translate, it gives you time to choose your words. "Your brothers and sisters in the south have food and shelter for your family. If you want to go, I can take you there."`
+			`	She hesitates but then the baby starts to cry. She looks at you with tears in her eyes and nods. You send some crew with her to help her pack her things and bring them to the ship.`
+				accept
+			
+			label "business"
+			`	The Unfettered mother never notices you as you pass by. You see the family later in a long queue for food rations.`
+				decline
+
+			label "condolences"
+			`	You tell her you are sorry for her loss. You see anger flare across her features as you turn to leave and if she wasn't holding an infant she might well have attacked you. "What good are your words monkey-man?" she shouts. " Will they feed my children? Did you come all this way to make fun of us? We have to choose between dying for a new world or starving on this one. Go back through the wormhole and give your words to your own kind!"`
+			`	A group of Unfettered roughly screen you off from the distraught mother and tell you to go to leave. She continues screaming at you and crying as you walk away.`
+				decline
+
+	on complete
+		payment 300000
+		conversation
+			`You are suprised to see a military honor guard in addition to what you assume are social services. They greet the family with a keening trill that your crewmember says is a sound of mourning. The Hai move in close and surround the group in what seems to be a group hug and then they move together into the spaceport. One of the Hai soldiers approaches you and says, "Thank you for brining our family back together" and gives you your payment of <payment>.`
+
+
+
+mission "Unfettered: Hunger Stories 2"
```

Comment:
I don't mind this one but I'm not sure why it happens only during an hunger situation?